js/flightlog_parser.js

```diff
@@ -727,13 +758,21 @@ var FlightLogParser = function(logData) {
                 that.sysConfig.magPID = parseCommaSeparatedString(fieldValue,3); //[parseInt(fieldValue, 10), null, null];
             break;
 
+            case "ff_weight":
+                // Add it to the end of the rollPID, pitchPID and yawPID
+                var ffValues = parseCommaSeparatedString(fieldValue);
+                that.sysConfig["rollPID"].push(ffValues[0]);
+                that.sysConfig["pitchPID"].push(ffValues[1]);
+                that.sysConfig["yawPID"].push(ffValues[2]);
+            break;
             case "feedforward_weight":
                 // Add it to the end of the rollPID, pitchPID and yawPID
                 var ffValues = parseCommaSeparatedString(fieldValue);
                 that.sysConfig["rollPID"].push(ffValues[0]);
                 that.sysConfig["pitchPID"].push(ffValues[1]);
                 that.sysConfig["yawPID"].push(ffValues[2]);
             break;
```

Comment:
Here happens the same than with `ff_transition`: two variables that are the same. You must "translate" one into the other and let only one.
The only difference, I suppose that by error, we don't have a "default" value in the original code for it.

Reply:
Fixed in the same way as above, appears to work correctly on testing with 4.3 and 4.2